Libraries/LibJS/Interpreter.cpp

```diff
@@ -299,7 +301,7 @@ Value Interpreter::construct(Function& function, Function& new_target, Optional<
 
 Value Interpreter::throw_exception(Exception* exception)
 {
-#ifdef __serenity__
+#if defined(__serenity__) and defined(INTERPRETER_DEBUG)
```

Comment:
Let's remove the `__serenity__` definition check, I only added that as it's writing stdout and dbg output to the same terminal on the host, but having this behind a single define is enough IMO.